src/Microsoft.ML.CodeGen/Templates/Console/PredictProgram.tt

```diff
@@ -0,0 +1,92 @@
+﻿<#@ template language="C#" linePragmas="false" #>
```

Comment:
https://github.com/dotnet/machinelearning/issues/4131 for removing generated .cs files